odoo/tools/misc.py

```diff
@@ -1637,3 +1637,13 @@ def hmac(env, scope, message, hash_function=hashlib.sha256):
         message.encode(),
         hash_function,
     ).hexdigest()
+
```

Comment:
I noticed you removed a blank line on base/models/res_partner - any reason you have an extra one here?